console-subscriber/src/lib.rs

```diff
@@ -222,7 +222,7 @@ impl TasksLayer {
 
         // Conservatively, start to trigger a flush when half the channel is full.
         // This tries to reduce the chance of losing events to a full channel.
-        let flush_under_capacity = config.event_buffer_capacity / 2;
+        let flush_under_capacity = config.event_buffer_capacity / 4;
```

Comment:
Conflicts with comment "half the channel is full".

To also help my mental model (feels like double negatives or something), this would now flush when there is only 1/4 of the capacity remaining. So the channel is 3/4 full. Right?

Reply:
(I believe Sean's reading is correct: it would now trigger a flush when channel is 3/4 full. But I also think Eliza is planning to remove this and some other unnecessary changes from this PR.)